docker/allPollerInOneContainer/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.16-alpine
+
+RUN apk add curl bash && apk add --virtual build-dependencies build-base gcc
+
+RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_x86_64 && \
+    chmod +x /usr/local/bin/dumb-init
+
+ARG INSTALL_DIR=/opt/harvest
+ARG BUILD_DIR=/opt/home
+ARG VERSION=2.0
+ARG RELEASE=nightly
+
+RUN echo $VERSION
+RUN echo $RELEASE
+
+# Set the Current Working Directory inside the container
+WORKDIR $BUILD_DIR
+
+RUN mkdir -p $INSTALL_DIR
+
+COPY . .
+
+RUN make VERSION=$VERSION RELEASE=$RELEASE
+
+RUN cp -a $BUILD_DIR/harvest.example.yml $INSTALL_DIR/harvest.yml
+
+RUN cp -aR bin $BUILD_DIR/conf $BUILD_DIR/docs $BUILD_DIR/grafana $BUILD_DIR/docker/allPollerInOneContainer/docker-entrypoint.sh $INSTALL_DIR && \
+   chmod +x $INSTALL_DIR/docker-entrypoint.sh
+
+VOLUME     [ "/opt/harvest" ]
+VOLUME     [ "/var/log/harvest" ]
+WORKDIR $INSTALL_DIR
+
+ENTRYPOINT ["/usr/local/bin/dumb-init", "--", "./docker-entrypoint.sh"]
```

Comment:
a multistage build that does not include the go tools will be significantly smaller - i happen to have an alpine build handy and du shows `152MB` for harvest

Reply:
this one is 800MB+ primary due to apk add actions.

Reply:
Thanks. Updated with multi stage builds. Size is now 155 MB